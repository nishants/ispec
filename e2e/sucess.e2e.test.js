const runComand  = require('./helpers/runCommand');

test("should run a single file in a project", async () => {
  const actual = await runComand("npm run e2e:test")
  expect(actual).toMatchSnapshot();
});