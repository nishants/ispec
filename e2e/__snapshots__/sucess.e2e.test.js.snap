// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should run a single file in a project 1`] = `
Object {
  "errors": Array [],
  "output": Array [
    "
> ispec@0.0.1 e2e:test /Users/dawn/projects/ispec
> npm --prefix ./example  run e2e

",
    "
> example@1.0.0 e2e /Users/dawn/projects/ispec/example
> ispec ./e2e

",
    "Running for dir  [
  '/Users/dawn/.nvm/versions/node/v14.5.0/bin/node',
  '/Users/dawn/.nvm/versions/node/v14.5.0/bin/ispec',
  './e2e'
]
",
  ],
  "status": Object {
    "code": 0,
    "error": undefined,
    "signal": null,
  },
}
`;
